lxd/instance/drivers/driver_qemu.go

```diff
@@ -8929,6 +9035,24 @@ func (d *qemu) checkFeatures(hostArch int, qemuPath string) (map[string]any, err
 		features["vhost_net"] = struct{}{}
 	}
 
+	// Get the host CPU model.
+	model, err := monitor.QueryCPUModel("kvm64")
```

Comment:
@mihalicyn I'd like to understand the decision for `kvm64` here (and document it) as looking into the `query-cpu-model-expansion` command it seems it can also take variants like:

```
x86 host                  processor with all supported host features 
x86 max                   Enables all features supported by the accelerator in the current host
```

and with kvm64 it has a variant too:

```
x86 kvm64                 (alias configured by machine type)
x86 kvm64-v1              Common KVM processor
```

Reply:
> I'd like to understand the decision for `kvm64` here (and document it) as looking into the

I think we should use `qemu64` as a baseline, because `kvm64` is considered as an obsolete and only there for compatibility reasons. While, internally there is not so much difference between `kvm64` and `qemu64` (from what I see in the QEMU code).

Reply:
ack thanks, useful links.

I wonder if we should also forcefully enable the spectre mitigation flags too?

Reply:
> I wonder if we should also forcefully enable the spectre mitigation flags too?

yes and no, because some of these mitigations can be useless for non-vulnerable AMD processors.

It worth experimentation what happen if we non-uniformly (across cluster members) enable mitigation flags on one node and do not enable on others (like if we have a mixed AMD / Intel cluster).

Reply:
Definitely worth experimenting with as we dont want to launch spectre vulnerable VMs.

Reply:
Also, I've got some updates regarding the `kvm64` choice. While from the CPU features perspective `kvm64` and `qemu64` are quite close, in practice, as Stéphane has told me, they have seen some weird effects with some user space programs which for some reason, consider this "QEMU Virtual CPU" as a sign of absence of hardware acceleration for virtualization. `kvm64` has a name like `Common KVM processor`. So, we can stick with `kvm64` for now.